CIP-0165/namespaces/README.md

```diff
@@ -0,0 +1,18 @@
+# Namespaces
+
+This is directory of the supported namespaces.
+
+Each namespace defines a non-intersecting slices of the data.
+
+| Shortname | Content                         |
+| --------- | ------------------------------- |
+| utxo      | UTXOs                           |
+| stake     | Stake delegation                |
+| rewards   | Reward accounts                 |
+| params    | Protocol parameters             |
+| pots      | Accounting pots (reserves etc.) |
+| spo       | SPO state                       |
```

Comment:
It's a nit pick. SPO is a Stake Pool "Operator", who is a human or a company, while Stake Pool is the actual entity that's running the node.
```suggestion
| stake_pools       | Stake Pools State                       |
```